autoRSA.py

```diff
@@ -294,6 +294,13 @@ async def on_ready():
                 os._exit(1)  # Special exit code to restart docker container
             await channel.send("Discord bot is started...")
 
+        # Process the message only if it's from the specified channel
+        @bot.event
+        async def on_message(message):
+            channel = bot.get_channel(DISCORD_CHANNEL)
```

Comment:
Because `DISCORD_CHANNEL` is already a channel ID, you can omit this line and just directly compare `message.channel.id == DISCORD_CHANNEL` instead.